compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -1,11 +1,11 @@
 import wso2/nballerina.err;
 
 function parseStmtBlock(Tokenizer tok) returns Stmt[]|err:Syntax {
-    Token? cur = tok.current();
-    if cur == "{" {
+    Token? firstToken = tok.current();
+    if firstToken == "{" {
```

Comment:
can be inlined. But I don't understand what this is an error in beta4.

Reply:
Since we cannot assign value to the variable which is narrowed outside the loop. Please check https://github.com/ballerina-platform/ballerina-lang/pull/32293

Reply:
Changed